tests.py

```diff
@@ -454,6 +454,39 @@ def test_fit2():
     assert not (tpot_obj._start_datetime is None)
 
 
+def test_imputer():
+    """Assert that the TPOT fit function will run without error in a dataset where NaNs are present."""
+    tpot_obj = TPOTClassifier(
+        random_state=42,
+        population_size=1,
+        offspring_size=2,
+        generations=1,
+        verbosity=0,
+        config_dict='TPOT light'
+    )
+    features_with_nan = np.copy(training_features)
+    features_with_nan[0][0] = float('nan')
+
+    tpot_obj.fit(features_with_nan, training_classes)
```

Comment:
In my merged code cleanup PR I removed those assertions as I considered them redundant.

`assert True` has no effect on the behavior of the test, and

```python
except ValueError:
    assert False
```

is just trading a `ValueError` for an `AssertionError`. Since no additional functionality is provided, and the `try ... except` pattern is just used to hint at what the test is checking, it should suffice to stick a comment next to the function call. I already include that information in the test function's docstring, but it could be repeated.